extensions/machine-learning-services/src/test/modelManagement/deployedModelService.test.ts

```diff
@@ -367,16 +376,17 @@ describe('DeployedModelService', () => {
 		testContext.config.setup(x => x.registeredModelTableName).returns(() => 'ta[b]le');
 		testContext.config.setup(x => x.registeredModelTableSchemaName).returns(() => 'dbo');
 		const expected = `
-		UPDATE [dbo].[ta[[b]]le]
-		SET
-		name = 'title1',
-		version = '1.1',
-		description = 'desc1'
-		WHERE artifact_id = 1`;
-		const actual = service.getUpdateModelQuery(dbName, model);
+		Insert into [dbo].[ta[[b]]le]
+		(artifact_name, group_path, artifact_content, name, version, description)
+		values (
+			'name1',
+			'ADS',
+			,
```

Comment:
line 384 empty value - intended?

Reply:
this is in the test. content is empty here/ won't happen other than test

---
In reply to: [406536133](https://github.com/microsoft/azuredatastudio/pull/9869#discussion_r406536133) [](ancestors = 406536133)